site/content/docs/5.1/components/orange-footer.md

```diff
@@ -7,31 +7,455 @@ aliases: "/docs/5.1/components/orange-footer/"
 toc: true
 ---
 
-## Overview
+## How it works
 
-{{< callout warning >}}
-### Orange footer is not a component yet
+Orange footer is basically a `<footer>` tag with a `.bg-dark` class. It can be composed of several parts:
+- mailing list bar
+- social media bar
+- navigation bar
+- service bar
+- **mandatory** bar
 
-The feature will be delivered with [#892]({{< param repo >}}/issues/892).
+## Examples
 
-In the meantime, some examples have been developed in the [footers examples section]({{< docsref "/examples/footers" >}}) by using mostly Boosted utilities and our Collapse JavaScript plugin.
+<footer role="contentinfo" class="bg-dark">
+  <h2 class="visually-hidden">Site map & informations</h2>
+  <!-- Mailing list -->
+  <div class="container footer-mailing">
+    <h3>Sign up to our mailing list</h3>
```

Comment:
as commented in the scss part, maybe here the use of h2/h3 could be informative, and document the fact that the project NEED to adapt it to his website title hierarchy. give sample with and without Hx tag could be usefull